.changeset/tricky-fans-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spectrum-web-components/tabs': patch
+---
+
+Added @spectrum-web-components/action-button dependency in Tabs since it uses in the direction button as its direct dependency
```

Comment:
```suggestion
Added `@spectrum-web-components/action-button` as a dependency for Tabs as its used in the direction button.
```

Some word-smithing here and adjusting it to be a complete sentence.

Reply:
@Rajdeepc can we add this in before merge?